systemd/systemd.go

```diff
@@ -326,6 +326,14 @@ const (
 	EmulationModeBackend
 )
 
+type MountUpdateStatus int
+
+const (
+	MountClean MountUpdateStatus = iota
```

Comment:
These values need some explanatory comments, as it's not clear to me what "MountClean" means.

Reply:
If I understand correctly the code, MountClean could be renamed to MountUnchanged, and and the comment should probably say that this does not guarantee that the mount exists and that it literally means that the preexisting situation was unchanged.

Maybe it could say that if MountClean is returned with a `nil` error then it means that the mount exists.

Reply:
I think `MountUnchanged` is better, yes.

The value returned when the error is not `nil` is relevant. It is the same as the mount unit name being an empty string.